lib/signaling/v2/room.js

```diff
@@ -190,14 +200,14 @@ class RoomV2 extends RoomSignaling {
   /**
    * @private
    */
-  _getOrCreateTrackReceiverDeferred(id) {
+  _getOrCreateTrackReceiverDeferred(id, idOrMid = 'id') {
     const deferred = this._trackReceiverDeferreds.get(id) || defer();
```

Comment:
is id going to be unique even when its `mid`? I suggest prepend it when maintaining those.
```suggestion
    const deferred = this._trackReceiverDeferreds.get(propertyName + ":" + propertyValue) || defer();
```

Reply:
MIDs will be unique.